modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/SourceEditor.soy

```diff
@@ -64,9 +64,8 @@
 				</svg>
 			</span>
 
-			{call liferay.frontend.Tooltip.render}
+			{call ClayTooltip.render}
```

Comment:
How can this work? ClayTooltip does not contain `selector` and `title` parameters. I'm looking at https://github.com/liferay/clay/blob/2.x/packages/clay-tooltip/src/ClayTooltip.soy